pages/k8s/tigera-secure-ee.md

```diff
@@ -44,7 +44,7 @@ Before you start, you will need:
 To start, deploy **Charmed Kubernetes** with Tigera Secure EE:
 
 ```bash
-juju deploy cs:~containers/kubernetes-tigera-secure-ee
+juju deploy containers-kubernetes-tigera-secure-ee
```

Comment:
because that's the charm.  we want the [bundle](https://charmhub.io/containers-kubernetes-tigera-secure-ee) for this particular blurb.

Reply:
okay, because that isn't confusing in the slightest at all. Can the bundle not go in ch? And can it not be called "something something bundle"?

Reply:
negative. the only proper bundles we want in CH are `charmed-kubernetes` and `kubernetes-core`.  Everything else should come from overlays, but we don't have an overlay yet for tigera (because we don't have a license to test tigera).  So for now, `containers-kubernetes-tigera-secure-ee` is the ch representation of `cs:~containers/kubernetes-tigera-secure-ee`, and it's all we've got.

Reply:
and I say "something something" because tigera-secure-ee isn't what it is called now anyway, which is another thing we should fix